### DISCUSSION TITLE
Mistake in the language tour?

### DISCUSSION THREAD

**Discussion by Tactile-Taco** [hylo-lang/hylo#825] [Q&A] [answered]
`subscript min_inout(_ x: inout Int, y: inout Int): Int {
  inout { if y > x { &x } else { &y } }
}

public fun main() {
  var (x, y) = (1, 2)
  &min_inout[&x, &z] += 2 //<---------------------------------shouldn't this be &min_inout[&x,&y] += 2 ?
  print(x) // 3
}`

**Comment by kyouko-taiga** [comment, marked as answer]
Yep, good catch.
